test/flex.html

```diff
@@ -18,17 +18,13 @@
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script src="../../web-component-tester/browser.js"></script>
-    <script src="../../test-fixture/test-fixture-mocha.js"></script>
 
     <link rel="import" href="../../iron-flex-layout/iron-flex-layout-classes.html">
-    <link rel="import" href="../../test-fixture/test-fixture.html">
     <link rel="import" href="../iron-collapse.html">
-
-    <style is="custom-style" include="iron-flex">
-    </style>
-
   </head>
   <body>
+    <custom-style><style is="custom-style" include="iron-flex">
```

Comment:
after discussing this offline, we concluded there's no harm in keeping this where it was (in `head`), so I'll just move it back there to have less changes in this PR 👌